### PR TITLE
Fix integration test

### PR DESCRIPTION
**Pull request by jeffhappily**
Note: Documentation fixes for https://docs.haskellstack.org/en/stable/ should target the "stable" branch, not master.

Please include the following checklist in your PR:

* [ ] Any changes that could be relevant to users have been recorded in the ChangeLog.md
* [ ] The documentation has been updated, if necessary.

Please also shortly describe how you tested your change. Bonus points for added tests!

### CODE REVIEW

File: snapshot-lts-11.yaml [hunk 1]
```diff
@@ -25,7 +25,9 @@ packages:
 - typed-process-0.2.6.0@rev:0 # for rio-0.1.10.0
 - rio-prettyprint-0.1.0.0@rev:0
 - hi-file-parser-0.1.0.0@rev:0
-- http-download-0.1.0.0@rev:0
+# - http-download-0.1.0.0@rev:0
```
Comment (snoyberg) [inline snapshot-lts-11.yaml]: I've released 0.2.0.0 to Hackage now

File: src/Stack/New.hs [hunk 2]
```diff
@@ -157,24 +157,26 @@ loadTemplate name logIt = do
...
-            dReq = DownloadRequest
-              { drRequest = req
-              , drHashChecks = []
-              , drLengthCheck = Just downloadFileSize
-              , drRetryPolicy = drRetryPolicyDefault
-              }
+        let path' = toFilePath path
+        exist <- liftIO $ fileExist path'
+
+        downloadFileSize <- if exist
```
Comment (snoyberg) [inline src/Stack/New.hs]: Maybe I misunderstood our discussions before, but why do we care about the size of the file on the filesystem?

Comment (jeffhappily) [reply src/Stack/New.hs]: The file size is also being use here https://github.com/commercialhaskell/stack/blob/cba6996791c2a7253388acab1b056aa5d30fe35e/src/Network/HTTP/StackClient.hs#L177 as the total size for the progress, I was thinking  to not change the behavior

Comment (snoyberg) [reply src/Stack/New.hs]: But isn't that known to be wrong?

Comment (jeffhappily) [reply src/Stack/New.hs]: Yeah that's true, OK I'll remove it

**Pull request merged**